.gitignore

```diff
@@ -7,7 +7,7 @@ workspace.xml
 *.iml
 .env.*
 .env
-dist/
+/dist
```

Comment:
leading slash means only dist in the root folder, but not all other dist folders, is that expected?

Reply:
I can add a more strict rule to allow every gh custom action  pushing a dist